src/inmanta/module.py

```diff
@@ -611,6 +649,19 @@ def __init__(self, local_repo: "ModuleRepo", remote_repo: "ModuleRepo") -> None:
         self.local_repo: ModuleRepo = local_repo
         self.remote_repo: ModuleRepo = remote_repo
 
+    def log_pre_install_information(self, project: "Project", module_name: str) -> None:
+        local_repo = self.local_repo.path_for(module_name)
+
+        assert local_repo is not None, f"Failed to install module {module_name} because its local repository is not defined."
```

Comment:
If the V1 module is not present on the local filesystem yet, this check will fail right? Or am I missing something?

Reply:
(I can't remember if we talked about this during one of our calls)
I'm assuming that after the install the v1 module should be present on the local filesystem, is it ok to do so ?